tools/build_env/build_env.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -eu
```

Comment:
A pattern in our shell scripts has developed over in the uproxy-docker repo, you might consider using it: it uses `getopts` and offers help and could help here (I ran the script several times, wondering why I saw no output, before I read the README a little closer):
https://github.com/uProxy/uproxy-docker/blob/master/testing/run-scripts/run_cloud.sh#L25